tools/src/versioning/android/expoModulesTransforms.ts

```diff
@@ -82,6 +82,16 @@ export function expoModulesTransforms(module: string, abiVersion: string): FileT
         },
       ],
     },
+    'expo-image': {
+      content: [
+        {
+          // glide only supports single AppGlideModule, we use the one from unversioned expo-image
+          paths: 'SourceMap.kt',
+          find: new RegExp(`\\bimport ${abiVersion}\\.expo\\.modules\\.image\\.GlideOptions`, 'g'),
+          replaceWith: 'import expo.modules.image.GlideOptions',
+        },
+      ],
+    },
```

Comment:
Let's add an option not to include the AppGlideModule from the expo image and create a separate glide app for Expo Go. The `react-native-fast-image` for instance, has that option https://github.com/DylanVann/react-native-fast-image/blob/main/docs/app-glide-module.md. However, for now, we can leave it as it's.

Reply:
we had this code but i cannot get it working.
https://github.com/expo/expo/blob/75c7ead1353d98508129058290b1271c6272fcf5/packages/expo-image/android/build.gradle#L60-L63

on the other hand, if we exclude AppGlideModule from expo-image, where should we put the AppGlideModule across sdks? e.g. `android/expoview/main/java/host/exp/exponent/` as shared AppGlideModule for all sdks?

Reply:
I'm not sure which place will be the best, but I would put it in the non-versioned code to share that across SDKs - probably in the kernel.

Reply:
#21015 doesn't actually work. kotlin compiler doesn't respect sourceSets excludes 🤷‍♂️
https://github.com/expo/expo/pull/20889/commits/9b2bdc8790863238dda44e31de8cd47a84740c51 is the alternative fix.